### PR TITLE
Pasma 3/front end account creation

### PR DESCRIPTION
**Pull request by KarstenRoland**

### CODE REVIEW

**Review by doranj5121 — CHANGES_REQUESTED** [review]
Please remove main.css and main.js. Thx!

File: test/client/components/AccountCreationCard.test.jsx
Comment (PistolRcks) [inline test/client/components/AccountCreationCard.test.jsx:1]: Since the test isn't passing, you can feel free to remove the failing tests.

**Review by PistolRcks — CHANGES_REQUESTED** [review]

File: test/client/components/AccountCreationCard.test.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+beforeAll(() => {
+    jest.spyOn(window, "alert").mockImplementation()
+    jest.spyOn(console, "log").mockImplementation()
+    jest.spyOn(console, "error").mockImplementation()
+});
+
+describe("Tests for <AccountCreationCard/>", () => {
+    test("Initial render", () => {
+        expect(1).toBe(1) //! I HATE YOU JEST
```
Comment (PistolRcks) [inline test/client/components/AccountCreationCard.test.jsx:15]: lol

**Review by PistolRcks — APPROVED** [review]

**Review by doranj5121 — APPROVED** [review]
Looks very good! Impressive AccountCreationCard

**Pull request merged**